src/custom/connectors/index.ts

```diff
@@ -1,16 +1,110 @@
-import { ChainId } from '@uniswap/sdk'
+import { Web3Provider } from '@ethersproject/providers'
 import { InjectedConnector } from '@web3-react/injected-connector'
+import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
+import { WalletLinkConnector } from '@web3-react/walletlink-connector'
+import { PortisConnector } from '@web3-react/portis-connector'
 
-export {
-  NETWORK_CHAIN_ID,
-  fortmatic,
-  portis,
-  network,
-  walletconnect,
-  walletlink,
-  getNetworkLibrary
-} from '@src/connectors'
-
-export const injected = new InjectedConnector({
-  supportedChainIds: [ChainId.MAINNET, ChainId.ROPSTEN, ChainId.RINKEBY, ChainId.GÖRLI, ChainId.KOVAN, ChainId.XDAI]
+import { FortmaticConnector } from 'connectors/Fortmatic'
+import { NetworkConnector } from 'connectors/NetworkConnector'
+
+const SUPPORTED_CHAIN_IDS = process.env.REACT_APP_SUPPORTED_CHAIN_IDS
+const DEFAULT_CHAIN_ID = Number(process.env.REACT_APP_CHAIN_ID || '1')
+
+const FORMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY
+const PORTIS_ID = process.env.REACT_APP_PORTIS_ID
+
+export const NETWORK_CHAIN_ID: number = parseInt(process.env.REACT_APP_CHAIN_ID ?? '1')
+
+type RpcNetworks = { [chainId: number]: string }
```

Comment:
nit: could also be `NetworkConnectorArguments['urls']` (interface can be imported from `./NetworkConnectors.ts`

Reply:
yep good tip, although that's not exported and the definition is so simple, I think it makes it even less readable. Also, we are more aware of this definition changes in the library and we need to address something.